cypress/e2e/GUI/disables_input_field_customer_details.cy.js

```diff
@@ -0,0 +1,8 @@
+describe('Desativação do campo de entrada de texto na página de detalhes do cliente', () => {
+    it('Deve desativar o campo de entrada de texto na página de detalhes do cliente', () => {
+      cy.visit('/clientes');
+
+      cy.get(':nth-child(1) > :nth-child(1) > button').click();
```

Comment:
É difícil saber em que botão se está clicando ao ler este seletor.
Recomendo usar um seletor mais legível.
Que tal algo como o seguinte?

```suggestion
      cy.get('tbody tr').first().click();
```